### DISCUSSION TITLE
Possible enhancement; Print error on every nth request while retrying

### DISCUSSION THREAD

**Discussion by mailsanchu** [curl/curl#10775] [Ideas]

**Comment by bagder** [comment]
Please use more words so that we understand what you're talking about. Perhaps an example command line of how it would look like when used?

**Comment by mailsanchu** [comment]
Given the below command, this will continuously print the host not found error message. What I am proposing is to print the message every nth attempt so there will be less noise.
`curl  --retry 15 --retry-delay 5 --retry-all-errors -O www.me.ig/a`
